CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -381,7 +375,8 @@ end
 --! Perform 'shake fist' action.
 function Patient:shakeFist()
   if self.shake_fist_anim then
-    self:queueAction(ShakeFistAction(), 1)
+    self:queueAction(ShakeFistAction(), 4)
```

Comment:
There is exactly one use of this function, so not much need for having it as a function.

Reply:
Could you provide a suggestion for this code spot? Lets see if we can get this one in a tidier place and we can merge what we have since falling is still disabled and for debugging only.

Reply:
``` bash
$ ag shakeFist 

CorsixTH/Lua/entities/humanoids/patient.lua
377:      self:shakeFist()
399:function Patient:shakeFist()
```

Usage is thus at line 377, maybe change it to something like
``` lua
    if math.random(1, 5) == 3 and self.shake_fist_anim then
      self:queueAction(ShakeFistAction(), 4)
      self:interruptAndRequeueAction(current, 5)
    else
      self:interruptAndRequeueAction(current, 4)
    end
```

Reply:
This also helps in understanding the magic 4 :)

Reply:
> EDIT: Hard-coding the queue position in the middle of the action queue makes it quite unusable. If you want to keep this, pass the index as a parameter.

Does your suggestion resolve this too? Or should this till be factored.

Reply:
Nope, didn't fix that. I would have said so if it did.